src/app/_services/depublication.service.spec.ts

```diff
@@ -27,9 +27,12 @@ describe('depublication service', () => {
   }));
 
   it('should depublish a dataset', async(() => {
-    service.depublishDataset('123').subscribe((res) => {
-      expect(res).toEqual(true);
-    });
+    service
+      .depublishDataset('123')
+      .subscribe((res) => {
+        expect(res).toEqual(true);
+      })
+      .unsubscribe();
```

Comment:
I see 4 instances all in this file, and the compiler reports them as having no expectations because what's missing is the mock http event that activates the subscription.  The tick() is not needed here, as this is provided by the mockHttp test helper which completes immediately (synchronously).  Once the mock call is made the expectation will be run (and the compiler will stop complaining!)